src/components/Base/Base.tsx

```diff
@@ -90,6 +88,7 @@ export const BaseComponent = <TResourceKey extends keyof Resources = keyof Resou
   return (
     <BaseContext.Provider
       value={{
+        error,
         fieldErrors,
```

Comment:
We provide both the error and fieldErrors via context. They only get our UI alerts if they use the BaseUIComponent below.

Still need to figure out how to navigate this with fieldErrors that need to be applied to the form fields. Do we just leave it to the partner to wire those up? Or do we find a way to normalize all form related errors so they aren't getting them from both the validation schema and the fieldErrors

Additionally, we need to investigate more how to help this be more ergonomic so they're not forced to respect both error and fieldErrors